src/server/server.js

```diff
@@ -6,7 +6,19 @@ var app = express();
 var http = require('http').Server(app);
 var io = require('socket.io')(http);
 var SAT = require('sat');
-var sql = require ("mysql");
+var pg = require ('pg');
+var localStorage = require ('node-localstorage');
+
+//Establish database connection
+pg.defaults.ssl = true;
```

Comment:
This seems quite arbitrary, why not make it a config option?

Reply:
Agreed !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!